src/couch_log/src/couch_log_writer_file.erl

```diff
@@ -38,6 +38,7 @@
 
 -ifdef(TEST).
 -compile(export_all).
+-compile(nowarn_export_all).
```

Comment:
Most of the functions from this module are private. I think it would be better to convert all `compile(export_all).` to explicit `-import([]).` of the functions which really need to be exported. I'll mark such places as well just so I wouldn't forget.

Reply:
I agree, and thanks for pointing out where it's appropriate to do so. In this case, it looks like we can simply replace the `export_all` with just
```
-export([
    maybe_reopen/1
]).
```